archlinuxcn/llvm-git/PKGBUILD

```diff
@@ -99,7 +99,7 @@ build() {
         -D LLVM_ENABLE_PROJECTS="lldb;polly;compiler-rt;lld;clang-tools-extra;clang" \
         -D TENSORFLOW_C_LIB_PATH="/usr/" # Force on
 
-    ninja -C _build LLVMgold all ocaml_doc
+    ninja -j8 -C _build LLVMgold all ocaml_doc
```

Comment:
And does it run out of memory or something without this on the buildbot?

Reply:
no, it's a local thing that i made a bit of a mistake on